src/python/pants/bin/local_pants_runner.py

```diff
@@ -286,22 +283,27 @@ def _compute_final_exit_code(*codes):
     return max_code
 
   def _run(self):
-    try:
-      self._maybe_handle_help()
+    self._set_start_time(time.time())
```

Comment:
Not allowing the start time to be injected via environment variable anymore sounds like something we would want to note in the PR description perhaps.